### PR TITLE
Existing enrollment actions

### PR DESCRIPTION
**Pull request by winterj**
Made enrollments polymorphic with classdays and dates, rather then storing the ID on the enrollment. For editing, it made the most sense to keep it per-enrollment, rather then treating the "group" of enrollments the same, since a student could take Monday for a few months and then start taking Thursday as well.

### CODE REVIEW

File: app/Enrollment.php [hunk 1]
```diff
@@ -4,19 +4,48 @@
...
+    } else {
+      return null;
     }
+      
+  }
+
+  public function otherEnrollmentsInClass(Enrollment $enrollment)
+  {
+    if ($enrollment->enrollable->class->season->SeasonType == 1) {
+      $otherEnrollments = DB::table('class_days')
```
Comment (Hobstweedl) [inline app/Enrollment.php]: I don't like this. Try to either do something with Eloquent, or wrap all of this logic up into 1 function in the model and call it.

File: app/Http/Controllers/EnrollmentsController.php [hunk 2]
```diff
@@ -57,7 +62,31 @@ public function show(Enrollment $enrollment)
      */
     public function edit(Enrollment $enrollment)
     {
-        //
+    $students = Student::getActive();
+    $families = Family::getActive();
+    $inactiveStudents = Student::getInactive();
+    $daysEnrolledInClass = DB::table('class_days')
```
Comment (Hobstweedl) [inline app/Http/Controllers/EnrollmentsController.php:68]: Same here

**Review by Hobstweedl — CHANGES_REQUESTED** [review]

**Comment by winterj** [comment]
Removed "other" enrollments stuff.

File: app/Http/Controllers/EnrollmentsController.php (hunk 2 above)
Comment (winterj) [reply app/Http/Controllers/EnrollmentsController.php:68]: Done and done.

**Review by Hobstweedl — APPROVED** [review]

**Pull request merged**